src/lib.rs

```diff
@@ -176,6 +181,37 @@ impl NotionApi {
         }
     }
 
+    pub async fn create_page(
+        &self,
+        parent: &Parent,
+        properties: &[PropertyValue],
+        children: &[Block],
+    ) -> Result<Page, Error> {
+        let new_page = PageCreate {
+            parent: parent.to_owned(),
+            properties: Properties {
+                properties: properties
+                    .iter()
+                    .map(|prop| (prop.as_id().to_string(), prop.to_owned()))
```

Comment:
What happens if the `PropertyValue` supplied doesn't have an identifier?

Reply:
Can you clarify what you mean by not having an identifier? PropertyValue guarantees that the property has its respective id.

Reply:
Oh! What if it doesn't have a property? Then that property is blank in the resulting row. I know this because my program just ignores some properties.

Reply:
I was more coming at a perspective of should the Caller have to know of the `PropertyValue`'s identifier before making the `create_page` call. This would require them to always have to make a GET request first to retrieve them in the case of a database.

Frankly, the identifiers for `PropertyValues` from Notion are a bit odd. Notion doesn't allow you to create multiple properties with the same name so the name is effectively the same as the identifier. The [API Spec for Create Page](https://developers.notion.com/reference/post-page)  even calls out that the key of the Map should be the Name OR identifier.

Reply:
In that case, I think maybe the Property thing should be refactored to store an alternative between the two possibilities, maybe? I'm not sure how to best approach it. 

One strategy would be a slight abuse: put the name in as the identifier value even though it's not an identifier really.

Reply:
I think where my head was at (from the other comment) was that the existing "response" models shouldn't be reused as "request" models.